README.md

```diff
@@ -368,294 +396,260 @@ A device renderer instance can be configured using the `options` argument (objec
 
-### `battery`
-
-<img align="right" src="./doc/assets/ic_battery.svg" alt="..."></img>
+#### `microphone`
 
 -   **Type:** `Boolean`
--   **Default:** `true`
--   **Compatibility:** `PaaS`, `SaaS`
+-   **Default:** `false`
+-   **Compatibility:** `PaaS`
 -   **Details:**
-    Enables or disables the battery widget. This widget can be used to set the battery level and state of the Android virtual device.
-
-### `gps`
+    Enables or disables microphone injection. This can be used to forward local microphone (or webcam audio) to the Android virtual device.
 
-<img align="right" src="./doc/assets/ic_location.svg" alt="..."></img>
+#### `mouse`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the gps widget. This widget can be used to set the gps location of the Android virtual device.
-    If you want to use a visual map instead of GPS coordinates number to set the location, you must import google maps
-    library with your API key.
-
-```html
-<!-- OPTIONAL: Import google maps library with your API key to enable map positioning feature -->
-<script src="https://maps.googleapis.com/maps/api/js?key=xxxxxxxxxxxxxxxxxxxxxxxx-yyyyyyyyyyyyyy"></script>
-```
+    Enables or disables the mouse events. If you want to disable all VM interaction, please also disable `touch` and `keyboard`.
 
-### `gpsSpeedSupport`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_nav_android_multiapp.svg"><img style="vertical-align: middle" src="./doc/assets/ic_nav_android_home.svg"><img style="vertical-align: middle" src="./doc/assets/ic_nav_android_back.svg"> `navbar`
 
 -   **Type:** `Boolean`
--   **Default:** `false`
+-   **Default:** `true`
+-   **Toolbar name:** `ButtonsEvents_RECENT_APP`, `ButtonsEvents_HOME`, `ButtonsEvents_BACK`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables gps speed support.
-
-### `capture`
+    Enables or disables the network widget. This widget can be used to enable or disable Wi-Fi or mobile network, and to set network throttling (mobile network type and signal strength) of the Android virtual device.
 
-<img align="right" src="./doc/assets/ic-screenshot_active_black.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_network.svg"> `network`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Network`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the capture widget. This widget can be used to capture the screen of the Android virtual device (screenshot or screencast).
-
-### `identifiers`
+    Enables or disables the network widget. This widget can be used to enable or disable Wi-Fi or mobile network, and to set network throttling (mobile network type and signal strength) of the Android virtual device.
```

Comment:
```suggestion
    Enables or disables the network widget. This widget can be used to enable or disable the Wi-Fi or mobile network, and to set network throttling (mobile network type and signal strength) of the Android virtual device.
```

Reply:
Looks like this correction can also be applied to the documentation of the `network` configuration.